### ISSUE TITLE
Search interface is accessible

### ISSUE THREAD

**Issue by rupachup** [silverstripe/silverstripe-ckan-registry#14]
## User story
As a sight impaired or keyboard only user
I can still use all the features of the register search
So I am included in benefiting from accessing information in the register.

## Acceptance criteria

- [ ] All form elements use best practice aria and WCAG acceptable approaches.
- [ ] The experience can be navigated and used from keyboard only
- [ ] When used with a screen reader it makes sense and provides any extra contextual information for a sight impaired user.
- [ ] Applied to the front end interface work only (we'll take the CMS as it stands).

### Notes:

- This could be part of an overall Definition of Done for the work.

**Comment by robbieaverill** [comment]
I suggest we close this - this should be an acceptance criterion for the "make a search form" ticket

**Comment by ScopeyNZ** [comment]
Yep these are really more to capture the "Definition of Done".
